Makefile

```diff
@@ -19,10 +19,12 @@ endif
 
 # Set platform for deps
 ifeq ($(OS_NAME), Linux)
+	PLATFORM = linux
 	JQ_PLATFORM = linux64
 	YQ_PLATFORM = linux
 	TRDL_PLATFORM = linux
```

Comment:
We have common PLATFORM and ARCH values and a couple of special cases: werf and jq. Why should we treat every tool as a special case?